src/learn/concepts/accounts.md

```diff
@@ -1 +1,48 @@
-# Accounts
+---
+Author: [Cedric](syyanyuhui@gmail.com)
+LastUpdate: 2024-05-06 
+---
+
+## Address Types in Aleo
+
+In Aleo, there are two types of address: user address and program address.
+
+User address have unique corresponding ViewKey and PrivateKey. In the Aleo network, we can create programs with privacy protection, where interactions between users and programs are invisible to everyone else. The **privacy of user addresses** is provided by the ViewKey, meaning whoever possesses the ViewKey corresponding to a user address has visibility over that address. The PrivateKey serves as proof of **ownership of the user address**; whoever possesses the PrivateKey corresponding to a user address owns all assets associated with that address. Address, ViewKey, and PrivateKey form a closely related set of mathematical objects designed based on cryptography. PrivateKey can derive ViewKey, and ViewKey can derive Address. This process is one-way; we cannot reverse-engineer ViewKey from Address or derive PrivateKey from ViewKey.
+
+[Learn more details about how Address、ViewKey、PrivateKey work](TODO: add link to AVM)
+
+Program address are derived from ProgramID through a secure hash algorithm. Unlike user address, program address do not have corresponding ViewKeys or PrivateKeys. More accurately, while there theoretically exist ViewKey and PrivateKey uniquely determined by program address, no one can derive the corresponding ViewKey and PrivateKey from a program address. Since program address lack PrivateKey and ViewKey, the assets associated with program addresses are entirely controlled by the logic defined within the program. This is crucial for building decentralized apps (Dapps) on the Aleo network to protect the assets stored by users within Dapps.
```

Comment:
- The Program address is derived from the ProgramID through a secure hash algorithm.
- Unlike the user address, the program address does not have corresponding ViewKeys or PrivateKeys
- Since the program address lack PrivateKey and ViewKey,